Tensile/SolutionStructs.py

```diff
@@ -2443,6 +2443,11 @@ def isDirectToLdsDoable(state, tc):
     #TN
     # use for all precisions with TransposeLDS=1
 
+    numRegisters = state["ProblemType"]["DataType"].numRegisters()
+    if numRegisters * state["GlobalReadVectorWidth"] != 1:
```

Comment:
Probably, it is better to use state["GlobalLoadVectorWidth%c"%tc].
GlobalReadVectorWidth might be different from GlobalLoadVectorWidthA/B.

Reply:
Updated in latest commit